pkg/sql/pgwire/conn_test.go

```diff
@@ -1610,11 +1617,16 @@ func TestParseClientProvidedSessionParameters(t *testing.T) {
 			wg.Add(1)
 			go func(query string) {
 				defer wg.Done()
-				ctx, cancel := context.WithTimeout(context.Background(), 100*time.Millisecond)
+				ctx, cancel := context.WithTimeout(context.Background(), 500*time.Millisecond)
 				defer cancel()
 				url := fmt.Sprintf("%s&%s", baseURL, query)
 				c, connErr := pgx.Connect(ctx, url)
 				if connErr != nil {
+					// If we didn't get the expected error from the server, then it
+					// means there was a connection failure.
+					if !testutils.IsError(connErr, "received connection") {
```

Comment:
Nit: maybe better would be `var ReceivedConnectionError = errors.New("received connection")`, then `errors.Is(connErr, ReceivedConnectionError)`. This also seems fine.

Reply:
to avoid breaking TestConn, i changed this to instead send a "connection successful" response